### PR TITLE
[9.0][NEW] account_bank _statement_import_move_line

### PR DESCRIPTION
**Pull request by luismontalba**
Account Bank Statement Import Move Line
==================================

This module adds a button to bank statement form to open a wizard to allow filtration, selection and importation of lines form journal items into the bank statement.

Usage
--------

* Go to Invoicing > Dashboard.
* Create a new bank statement from a bank journal.
* Click button "Import Journal Items".
* Select filtering options.
* Click on button "Add All Move Lines" to auto-select the move lines matching
 the selected criteria or click on Add an item to manually select the move
 lines filtered by the criteria.
* Click on button "Create Statement Lines".

cc @Tecnativa

### CODE REVIEW

File: account_bank_statement_import_move_line/README.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=======================================
+Account Bank Statement Import Move Line
+=======================================
+
+This module adds a button to bank statement form to open a wizard to allow
+filtration, selection and importation of lines form journal items into the
```
Comment (pedrobaeza) [inline account_bank_statement_import_move_line/README.rst]: s/filtration/filtering

Comment (pedrobaeza) [reply account_bank_statement_import_move_line/README.rst]: s/form/from

File: account_bank_statement_import_move_line/README.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+Usage
+=====
+
+#. Go to Invoicing > Dashboard.
+#. Create a new bank statement from a bank journal.
+#. Click button "Import Journal Items".
+#. Select filtering options.
+#. Click on button "Add All Move Lines" to auto-select the move lines matching
+ the selected criteria or click on Add an item to manually select the move
```
Comment (pedrobaeza) [inline account_bank_statement_import_move_line/README.rst]: For second line and successive, you have to indent 2 spaces.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Little changes on code review

@chienandalu please make a functional review

**Comment by pedrobaeza** [comment]
Please check also Travis status

**Review by chienandalu — APPROVED** [review]
Functional review :+1:

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please port it also to v10.

File: account_bank_statement_import_move_line/README.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=======================================
+Account Bank Statement Import Move Line
+=======================================
+
+This module adds a button to bank statement form view to open a wizard to allow
+filtering, selecting and importing lines form journal items into the bank
```
Comment (alexis-via) [inline account_bank_statement_import_move_line/README.rst:10]: form -> from ?

Comment (luismontalba) [reply account_bank_statement_import_move_line/README.rst:10]: Form view (now clarified in the README)

**Comment by luismontalba** [comment]
Changes done and test added (commit squashed)

**Pull request merged**